deploy/base/profiles/security-profiles-operator.json

```diff
@@ -1,10 +1,7 @@
 {
   "defaultAction": "SCMP_ACT_ERRNO",
   "architectures": [
-    "SCMP_ARCH_X86_64",
-    "SCMP_ARCH_X86",
-    "SCMP_ARCH_X32",
-    "SCMP_ARCH_AARCH64"
+    "SCMP_ARCH_PPC64LE"
```

Comment:
Combine the prior and this so you have 6 arches listed (including SCMP_ARCH_PPC64LE
SCMP_ARCH_S390X